### DISCUSSION TITLE
How to get provider in react native

### DISCUSSION THREAD

**Discussion by Haris565** [Web3Auth/Web3Auth#425] [Help]
I implemented the login flow and also getting the desired fields in response but how can i sign the messages and get provider in react native

**Comment by BboyStatix** [comment]
Hi there, currently none of the mobile SDKs return a provider (unlike the web sdks which can be referenced [here](https://web3auth.io/docs/sdk/web/providers))

So what you will have to do is to use the private key directly to interact with the specific blockchain. The private key is returned in the [sample response](https://web3auth.io/docs/sdk/react-native/usage#sample-response)
